Render back button & breadcrumbs outside of main content

libuk opened the pull request:
# Changes in this PR
It was flagged that there was too much padding appeared on top of the back buttons. This is due to padding coming from the `govuk-main-wrapper` class on the `main` tag, which is the parent of the back link anchor.

To remedy this, this change renders the back link outside of the `main` tag as the breadcrumbs are technically for navigation and therefore doesn't need to be included in `main`. This is consistent with other gov uk pages[1].

The same fix is applied to breadcrumbs.

[1]
https://www.gov.uk/sign-in-universal-credit

## Screenshots of UI changes
### Back link - Before
<img width="229" alt="padding_referrals_start_before" src="https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/6377078/846c9fd3-f2c5-4fde-883d-9f14adf350ef">

### Back link - After
<img width="310" alt="padding_referrals_start_after" src="https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/6377078/fe6e79f9-b294-4bcd-a693-ea2d84322013">

### Breadcrumbs - Before
![padding_find_bedspace_before](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/6377078/5946b95f-216c-4d11-8ae9-e38bd49872ad)

### Breadcrumbs - After
![padding_find_bedspace_after](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/assets/6377078/0d01e773-7e69-4d8a-94f1-f29772aec8ae)

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

<!-- Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible. You can monitor deployment failures in CircleCI
itself and application errors are found in
[Sentry](https://ministryofjustice.sentry.io/issues/?project=4504129156218880&referrer=sidebar&statsPeriod=24h).
Both events should be automatically sent to our [Slack
channel](https://mojdt.slack.com/archives/C048BJS7S2F). -->

libuk committed:
Render back button outside of main content

It was flagged that too much padding appeared on top of the back
buttons. This is due to padding coming from the `govuk-main-wrapper`
class on the `main` tag, which is the parent of the back link anchor.

To remedy this, this change renders the back link outside of the `main`
tag as the breadcrumbs are technically for navigation and therefore
doesn't need to be included in `main`. This is consistent with other gov
uk pages[1].

[1]
https://www.gov.uk/sign-in-universal-credit

bryangaledxw commented:
Apologies if I'm jumping on this before it's ready - looks good! Think it's worth though talking to Barry/Selina about if we want to do the same for breadcrumbs at the same time? If you search "import breadCrumb" you should see where we're using them

libuk commented:
@bryangaledxw Ah my bad, I should've mentioned in the description that I spoke to Barry and the plan is to do the same to breadcrumbs in this PR.

libuk commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Remove padding from top of pages(s), so back button is directly underneath header.](https://trello.com/c/tuhHjVwb/1412-remove-padding-from-top-of-pagess-so-back-button-is-directly-underneath-header)

libuk committed:
Render breadcrumbs outside of main content

It was flagged that too much padding appeared on top of the back
buttons and breadcrumbs.

We changed the back links in a previous commit, this commit applies the
same fix to the breadcrumbs.

bryangaledxw reviewed (approved):
Looks good!
